### ISSUE TITLE
[Feature Request] Include archivec chats in unread count

### ISSUE THREAD

**Issue by neesti** [telegramdesktop/tdesktop#7170]
Hello.
Please add this option to "Notifications" settings (Badge counter):
Include archived chats in unread count. 
![image](https://user-images.githubusercontent.com/14938202/74155297-9a0af180-4c29-11ea-8226-cb0f84e0f917.png)

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/6105

**Comment by maxoku** [comment]
Include archived chats? Aren't they already included? I've got badge with 1 unread chat and I couldn't find it on my list, so I looked into archived ones and then I found it. It would be better rather to exclude it, but the [#6105](https://github.com/telegramdesktop/tdesktop/issues/6105) idea allows to do it per single chat. Not completely the same, but I guess it's easier to just pick one idea and follow. 🤔
